WebDevStudios/wds-headless-wordpress
# Dependabot 2021-08-02

Related PR: https://github.com/WebDevStudios/nextjs-wordpress-starter/pull/582

### Link

WP:
https://nextjsdevstart.wpengine.com/wp-admin/

Frontend:
https://nextjs-wordpress-starter-develop.vercel.app/

### Description

This PR updates the following Composer dependencies:

```text
composer/xdebug-handler (2.0.2)
google/apiclient-services (v0.206.0)
justinrainbow/json-schema (5.2.11)
mck89/peast (v1.13.5)
symfony/polyfill-php80 (v1.23.1)
symfony/css-selector (v5.3.4)
symfony/polyfill-mbstring (v1.23.1)
symfony/polyfill-intl-grapheme (v1.23.1)
symfony/console (v5.3.6)
symfony/filesystem (v5.3.4)
symfony/finder (v5.3.4)
symfony/process (v5.3.4)
webdevstudios/gravityforms (2.5.8)
wp-cli/php-cli-tools (v0.11.13)
wp-cli/widget-command (v2.1.5)
wp-cli/search-replace-command (v2.0.14)
composer/composer (2.1.5)
wp-cli/i18n-command (v2.2.9)
wp-cli/export-command (v2.0.9)
wp-cli/db-command (v2.0.14)
wpackagist-plugin/gutenberg (11.1.0)
wpackagist-plugin/wp-graphql (1.6.1)
wpackagist-plugin/wp-search-with-algolia (2.0.1)
```

### Screenshots

WP Dashboard:
![screenshot](https://dl.dropbox.com/s/yx6qufotsta5s9a/Screen%20Shot%202021-08-02%20at%2010.31.46%20AM.png?dl=0)

Post Editor:
![screenshot](https://dl.dropbox.com/s/iqosg6jcrr18hhr/Screen%20Shot%202021-08-02%20at%2010.32.25%20AM.png?dl=0)

Next.js Frontend:
![screenshot](https://dl.dropbox.com/s/na7wpoaj1rbfjap/Screen%20Shot%202021-08-02%20at%2010.34.25%20AM.png?dl=0)

### Steps To Verify Feature

How will your Lead Engineer and/or stakeholder test this?

1. `gh pr checkout 50`
2. Verify all the plugins have updated
3. Go to the Next.js WordPress Starter --> `npm run build && npm start`
4. Verify the front-end builds successfully
